### PR TITLE
Removed Tile URI Workaround

### PR DESCRIPTION
**Pull request by OmarShehata**
Fixes https://github.com/AnalyticalGraphicsInc/cesium/issues/6868. 

I confirmed that this hasn't broken any tests, and the "3D Tiles Photogrammetry Classification" example in Sandcastle works with no errors. 

@lilleyse can you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @OmarShehata!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/remove-workaround/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by tfili** [comment]
Oh yeah, I remember putting this in. This was put in because we converted a bunch of data for customers and then switched to using Resource objects which did correct URL resolution.

I guess it’s been long enough to pull it. I’m assuming at this point those old datasets wouldn’t load anyway or if this breaks them, do we care?

File: Source/Scene/Cesium3DTileset.js [hunk 1]
```diff
@@ -666,12 +666,6 @@ define([
          * @default false
          */
         this.debugShowUrl = defaultValue(options.debugShowUrl, false);
-
-        // A bunch of tilesets were generated that have a leading / in front of all URLs in the tileset JSON file. If the tiles aren't
```
[the remark this answers is not in the document]
Comment (OmarShehata) [reply Source/Scene/Cesium3DTileset.js]: Good catch! Should be good now.

**Comment by lilleyse** [comment]
Works great, thanks!

**Pull request merged**